tasks/deployment/v3-data-availability-proposal.ts

```diff
@@ -0,0 +1,15 @@
+import { types } from 'hardhat/config';
+
+import mainnetAddresses from '../../src/deployed-addresses/mainnet.json';
+import { hardhatTask } from '../../src/hre';
+import { DIP_22_IPFS_HASH } from '../../src/lib/constants';
+import { createV3DataAvailabilityProposal } from '../../src/migrations/v3-data-availability-proposal';
+
+hardhatTask('deploy:v3-data-availability-proposal', 'Create proposal to fix data availability bug on v3 perp contract.')
+  .addParam('proposalIpfsHashHex', 'IPFS hash for the uploaded DIP describing the proposal', DIP_22_IPFS_HASH, types.string)
+  .addParam('governorAddress', 'Address of the deployed DydxGovernor contract', mainnetAddresses.governor, types.string)
+  .addParam('starkwarePriorityAddress', 'Address of the deployed short timelock Executor contract', mainnetAddresses.starkwarePriorityTimelock, types.string)
```

Comment:
Nit: mind updating the description to mention this is the deployed starkware priority executor contract?